README.md

```diff
@@ -1,24 +1,80 @@
-# Rust binding to [Open Quantum Safe][oqs]'s [liboqs][]
+liboqs-rust: Rust bindings for liboqs
+=====================================
 
-This crate provides bindings to [liboqs][]. It provides unsafe `ffi` bindings in the `oqs-sys` crate.
-Safe wrappers are offered via the `oqs` crate.
+[![Build status](https://github.com/dstebila/oqs-rs/workflows/Continuous%20integration/badge.svg)](https://github.com/dstebila/oqs-rs/actions?query=workflow%3A"Continuous+integration")
 
+**liboqs-rust** offers a Rust wrapper for the [Open Quantum Safe](https://openquantumsafe.org/) [liboqs](https://github.com/open-quantum-safe/liboqs/) C library, which is a C library for quantum-resistant cryptographic algorithms.
 
-## Adding new algorithms
+Pre-requisites
+--------------
 
-### KEMs:
+liboqs-rust depends on the [liboqs](https://github.com/open-quantum-safe/liboqs) C library; liboqs must first be compiled as a Linux/macOS/Windows library (i.e. using `ninja install` with `-DBUILD_SHARED_LIBS=ON` during configuration), see the specific platform building instructions below.
```

Comment:
This is not accurate. `oqs-sys` completely transparantly handles compiling ``liboqs`` using `cmake`. See `build.rs` in `oqs-sys`.

We could add some logic to `build.rs` to detect presence of `liboqs` somewhere else, but `oqs-sys` also needs the source code to that version — it generates the ``ffi`` interface by parsing the C headers.

Reply:
Can you put relevant text in here?